release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3340,6 +3340,43 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-14-35_{context}"]
+=== RHSA-2024:5433 - {product-title} 4.14.35 bug fix and security update
+
+Issued: 21 August 2024
+
+{product-title} release 4.14.35, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:5433[RHSA-2024:5433] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:5436[RHSA-2024:5436] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.35 --pullspecs
+----
+
+[id="ocp-4-14-35-updating-kubernetes-version_{context}"]
+===== Updating to Kubernetes v1.27.16
+
+* This release contains the updates resulting from an update to Kubernetes v1.27.16. (link:https://issues.redhat.com/browse/OCPBUGS-37623[*OCPBUGS-37623*])
+
```

Comment:
Feature backport in this release: 

These xrefs should build but won't resolve to the section heading until #80577 merges

```suggestion

[id="ocp-4-14-35-enhancements"]
==== Enhancements

The following enhancements are included in this z-stream release:

[id="ocp-4-14-35-enhancements-azure-res-cap_{context}"]
===== Configuring Capacity Reservation by using machine sets

* {product-title} release {product-version}.25 introduces support for on-demand Capacity Reservation with Capacity Reservation groups on {azure-full} clusters.
For more information, see _Configuring Capacity Reservation by using machine sets_ for xref:../machine_management/creating_machinesets/creating-machineset-azure.adoc#machineset-azure-capacity-reservation_creating-machineset-azure[compute] or xref:../machine_management/control_plane_machine_management/cpmso-using.adoc
#machineset-azure-capacity-reservation_cpmso-using[control plane] machine sets. (link:https://issues.redhat.com/browse/OCPCLOUD-1646[*OCPCLOUD-1646*])
```